bin/use-client-pgbouncer

```diff
@@ -65,6 +46,11 @@ function dbname() {
 function setup-client-pgbouncer() {
   local postgres_url
 
+  if is-enabled "${PGBOUNCER_VARS_DISABLED}"; then
+    inf "pgBouncer <VAR>_PGBOUNCER exports are disabled, skipping..."
+    return 0
+  fi
```

Comment:
Note to selves: If this functionality is baked into the buildpack then we no longer need it in the application code